src/agent/src/config.rs

```diff
@@ -183,9 +182,10 @@ impl AgentConfig {
             }
         }
 
-        if let Ok(value) = env::var(TRACE_TYPE_ENV_VAR) {
-            if let Ok(result) = value.parse::<tracer::TraceType>() {
-                self.tracing = result;
+        if let Ok(value) = env::var(TRACING_ENV_VAR) {
+            match value.as_str() {
```

Comment:
Should use `get_bool_value` to parse the config value like command-line way?

Reply:
Good catch @liubin! Updated.